Sources/UIViewControllerExtensions.swift

```diff
@@ -224,7 +224,12 @@ extension UIViewController {
     public func popVC() {
         _ = navigationController?.popViewController(animated: true)
     }
-
+    
+    /// EZSwiftExtensions
+    public func popToRootVC() {
+        _ = navigationController?.popToRootViewController(animated: true)
```

Comment:
I personally never use this values, so I think it's ok.

@piv199 how do you use this returned value?